pkg/tfvars/ovirt/ovirt.go

```diff
@@ -71,6 +78,50 @@ func TFVars(
 		}
 		cfg.BaseImageLocalFilePath = imageFilePath
 	}
-
+	if len(controlPlaneAffinityGroups) > 0 {
+		cfg.ControlPlaneAffinityGroups = createControlPlaneAffinityGroups(controlPlaneAffinityGroups, infraID)
+	}
+	cfg.ComputeAffinityGroups = createComputeAffinityGroups(
+		computeMachines, controlPlaneAffinityGroups, infraID)
 	return json.MarshalIndent(cfg, "", "  ")
 }
+
+func createControlPlaneAffinityGroups(masterAffiniryGroups []*ovirt.AffinityGroup, infraID string) []map[string]interface{} {
+	var agMap []map[string]interface{}
+	for _, ag := range masterAffiniryGroups {
+		if ag.Name != "" {
```

Comment:
Can we please fail if the user doesn't input a name for the AG?

Reply:
Failing here is good, but I would hope that this would have already been caught by validation.

Reply:
Currently, it is not, but I'll add that to the validations